tests/tools/test_betting_strategies.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import numpy as np
+from prediction_market_agent.markets.all_markets import omen
+from prediction_market_agent.tools.gtypes import (
+    Probability,
+    xdai_type,
+    wei_type,
+    usd_type,
+    HexAddress,
+    OmenOutcomeToken,
+    HexStr,
+    Wei,
+    xDai,
+)
+from prediction_market_agent.tools.betting_strategies import (
+    get_kelly_criterion_bet,
+    get_market_moving_bet,
+)
+
+TEST_OMEN_MARKET = omen.OmenMarket(
+    id=HexAddress(HexStr("0x76a7a3487f85390dc568f3fce01e0a649cb39651")),
+    title="Will Plex launch a store for movies and TV shows by 26 January 2024?",
+    collateralVolume=Wei(4369016776639073062),
+    usdVolume=usd_type("4.369023756584789670441178585394842"),
+    collateralToken=HexAddress(HexStr("0xe91d153e0b41518a2ce8dd3d7944fa863463a97d")),
+    outcomes=["Yes", "No"],
+    outcomeTokenAmounts=[
+        OmenOutcomeToken(7277347438897016099),
+        OmenOutcomeToken(13741270543921756242),
+    ],
+    outcomeTokenMarginalPrices=[
+        xdai_type("0.6537666061181695741160552853310822"),
+        xdai_type("0.3462333938818304258839447146689178"),
+    ],
+    fee=wei_type(20000000000000000),
+)
+
+
+@pytest.mark.parametrize(
+    "wanted_p_yes_on_the_market, expected_buying_xdai_amount, expected_buying_outcome",
+    [
+        (Probability(0.1), xdai_type(25.32), "No"),
```

Comment:
Where did you get these `expected_buying_xdai_amount`s from?

Reply:
Copied from aiomen website, if you fill something in, it will show you how the market will change 
![Screenshot by Dropbox Capture](https://github.com/evangriffiths/prediction-market-agent/assets/15619339/b3e71490-5bc9-4fff-84c0-8af7787f29db)